models/reporting/olids/data_quality/dq_eosinophil_percentage_issues.sql

```
@@ -32,15 +32,16 @@ SELECT
     CASE
         WHEN is_negative THEN 'Negative Value'
         WHEN is_extreme_outlier THEN 'Extreme Outlier (> 80%)'
-        WHEN confidence = 'NONE' AND conversion_reason = 'Excluded unit on this measurement type' THEN 'Excluded Unit (' || original_result_unit_code || ')'
-        WHEN confidence = 'NONE' THEN 'Non-Accepted Unit on Percentage Measurement'
+        WHEN inferred_value IS NULL THEN 'Excluded Unit (' || original_result_unit_code || ')'
```

SQLFluff (4.2.2)

[error] Boolean/null literals must be lower case.

(CP04)